docs/source/examples/update_dataset.rst

```diff
@@ -57,10 +54,49 @@ Commit the dataset:
 | Users can switch between different commits to use different version of labels.
 
 .. important::
-   | uploading labels operation will overwrite all types of labels in data.
+   Uploading labels operation will overwrite all types of labels in data.
 
 ***************
  Update Data
 ***************
 
-This part will be introduced later.
+| Add new data to dataset.
+| Set `skip_uploaded_files=True` to skip uploaded data.
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Updata data/ upload dataset"""
+   :end-before: """"""
+
+By default will overwrite uploaded data.
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Updata data/ overwrite dataset"""
+   :end-before: """"""
+
+.. note::
+   The segment name and data name are used to determine if is the same data.
```

Comment:
... identify data, which means if two data's segment name and dataset name are the same, then they will be regarded as one data.